plugin-gradle/src/main/java/appland/execution/AppMapGradleAgentRunner.java

```diff
@@ -34,11 +34,7 @@ public boolean canRun(@NotNull String executorId, @NotNull RunProfile profile) {
     @Override
     public void execute(@NotNull ExecutionEnvironment environment) throws ExecutionException {
         verifyJdk(environment);
-        try {
-            findDelegate().execute(environment);
-        } finally {
-            AppMapJvmExecutor.sendTelemetry(environment.getRunProfile());
-        }
+        findDelegate().execute(environment);
```

Comment:
Why is this no longer in a `try` block? If it raises an exception, is it a problem that we are no longer catching it?

Reply:
Try-finally means that exceptions are passed to the caller (i.e. they are not caught), but the finally code is always executed, even when an exception was thrown by the try block. 
we don‘t have code in finally anymore, so it was removed.

Reply:
Okay, makes sense, thank you!